rules/private/phases/phase_coverage_jacoco.bzl

```diff
@@ -59,14 +57,7 @@ def phase_coverage_jacoco(ctx, g):
         ),
     ])
 
-    return struct(
-        instrumented_files = struct(
-            dependency_attributes = _coverage_replacements_provider.dependency_attributes,
-            extensions = ["scala", "java"],
-            source_attributes = ["srcs"],
-        ),
-        replacements = replacements,
-    )
+    return struct(replacements = replacements)
```

Comment:
Do we no longer need `instrumented_files`?

Reply:
See my comment [here](https://github.com/lucidsoftware/rules_scala/pull/78#discussion_r1973891186). This was only used in `phase_coda`, but I removed that reference to it. Therefore, it was no longer needed.